contracts/tokenomics/maker/src/contract.rs

```diff
@@ -534,6 +546,14 @@ fn distribute(deps: DepsMut, env: Env, cfg: &mut Config) -> Result<Vec<SubMsg>,
                 to_governance_asset.into_msg(&deps.querier, governance_contract)?,
             ))
         }
+        pure_astro_reward -= pure_astro_reward.multiply_ratio(
+            Uint128::from(cfg.governance_percent.u64()),
+            Uint128::new(100),
+        );
+        current_preupgrade_distribution -= current_preupgrade_distribution.multiply_ratio(
+            Uint128::from(cfg.governance_percent.u64()),
+            Uint128::new(100),
+        );
```

Comment:
Why do you exclude governance portion?

Reply:
Just to show exact amount of distributed astro to staking contract. But this is the point for discussion should we display total distributed astro or just astro which could be then claimed from staking contract.

Reply:
Decided to return total distributed astro (including governance fee)